nautobot/core/api/fields.py

```diff
@@ -7,6 +7,7 @@
 from timezone_field.rest_framework import TimeZoneSerializerField as TimeZoneSerializerField_
 
 
+# TODO: why is this not a serializers.ChoiceField subclass??
```

Comment:
If I'm not mistaken it's because we allow grouped choices which `ChoiceField` wouldn't support.

Reply:
I can see below that we do support that, but I'd still expect that we'd want to extend ChoiceField and override things where necessary rather than have to implement it from the base `Field` class.